pgmax/fg/graph.py

```diff
@@ -924,6 +919,31 @@ def run_bp(
         if evidence_updates is not None:
             evidence = update_evidence(evidence, evidence_updates, bp_state.fg_state)
 
+        return BPArrays(
+            log_potentials=log_potentials, ftov_msgs=ftov_msgs, evidence=evidence
+        )
+
+    def bp_arrays_update(
```

Comment:
Calling this `run_bp` might be more appropriate. This is not any kind of update, but only updating the messages by running BP